File: bddtests/src/main/kotlin/mocking/tpp/models/LogOffReply.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "LogoffReply")
@XmlAccessorType(XmlAccessType.FIELD)
data class LogOffReply(
        @XmlAttribute var uuid: String = "default uuid"
)

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Tpp/Models/LogoffReply.cs
using System;
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Tpp.Models
{
    [Serializable]
    public class LogoffReply
    {
        [XmlAttribute("uuid")]
        public Guid Uuid { get; set; }
    }
}